scripts/build_golden_dataset.py

```
@@ -0,0 +1,472 @@
+        JOIN cre_node_links l ON l.node = n.id
+        JOIN cre c ON c.id = l.cre
+        WHERE n.section IS NOT NULL AND length(n.section) > 20
+          AND (n.name LIKE '%Top 10 2021%' OR n.name LIKE '%CWE%')
+        GROUP BY n.id
+        HAVING COUNT(DISTINCT c.external_id) BETWEEN 2 AND 4
+        ORDER BY n.name, n.section_id
+        LIMIT 15
+        """
+    ).fetchall()
+    out = []
+    for node_id, name, section_id, text, cre_concat in rows:
```

Ruff (0.15.17)

[warning] Loop control variable `node_id` not used within loop body

Rename unused `node_id` to `_node_id`

(B007)